cli/args/flags.rs

```diff
@@ -1,16 +1,8 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 
-use std::collections::HashSet;
-use std::env;
-use std::ffi::OsString;
-use std::net::SocketAddr;
-use std::num::NonZeroU32;
-use std::num::NonZeroU8;
-use std::num::NonZeroUsize;
-use std::path::Path;
-use std::path::PathBuf;
-use std::str::FromStr;
-
```

Comment:
Please revert import changes here...

Reply:
@bartlomieju 

It's not my changes.  It appears on my change after I updated the branch yesterday and resolves the conflict. 

![image](https://github.com/user-attachments/assets/a94ff19e-8e70-483a-bc4f-e581a9f4c083)
![image](https://github.com/user-attachments/assets/0e11fba3-9a50-4e14-92ee-4090471e9c13)

If I delete it, the build will fail since multiple variables and methods related to those imports been undefined.

So, if you like, I can rollback all the changes that cause conflict. ?

Reply:
done